src/main/kotlin/org/rust/cargo/toolchain/Cargo.kt

```diff
@@ -48,6 +49,32 @@ private val LOG = Logger.getInstance(Cargo::class.java)
  * because the user can always just `rm ~/.cargo/bin -rf`.
  */
 class Cargo(private val cargoExecutable: Path) {
+
+    data class Package(val name: String, val version: SemVer? = null) {
```

Comment:
This can be confused with `CargoWorkspace.Package`, consider rename it to something

Reply:
Fixed.

Reply:
Hmm, `crate` is too confusing to... Maybe BinaryPackage?

Reply:
Now I use `BinaryCrate` since this name is used in help info (`cargo install --help`):
> This command manages Cargo's local set of installed binary crates.